### ISSUE TITLE
After 'npm install' and 'npm run dev', should the react dev tools load in the inspector?

### ISSUE THREAD

**Issue by timothy-o-johnson** [pbarbiero/basic-electron-react-boilerplate#20]
Hello,
I've forked and completed an 'npm install' and 'npm run dev'.  Now the electron app pops up, but there's no React dev tool in the inspector.  Is this correct? I have a feeling that the server is not running.

Thanks,
Tim

![image](https://user-images.githubusercontent.com/13969159/38680129-8f8ec854-3e33-11e8-8475-7f3b7ad24b4d.png)

**Comment by lourencorodrigo** [comment]
Yes. This is correct. You can take a look at **[electron-devtools-installer](https://github.com/MarshallOfSound/electron-devtools-installer)**.
